frontend/src/pages/PropertyDetails.jsx

```
@@ -0,0 +1,813 @@
+            >
+                <div 
+                    className="relative w-[90%] h-[90%] max-w-[1200px] flex items-center justify-center"
+                    onClick={(e) => e.stopPropagation()}
+                    onKeyDown={(e) => {
+                        if (e.key === "ArrowLeft" && images.length > 1 && currentImageIndex > 0) {
+                            handlePrev();
+                        } else if (e.key === "ArrowRight" && images.length > 1 && currentImageIndex < images.length - 1) {
+                            handleNext();
+                        }
+                    }}
+                    tabIndex={0} // Allows this div to capture key events
```

Biome (1.9.4)

[error] The HTML element div is non-interactive. Do not use tabIndex.

Adding non-interactive elements to the keyboard navigation flow can confuse users.
Unsafe fix: Remove the tabIndex attribute.


(lint/a11y/noNoninteractiveTabindex)